### DISCUSSION TITLE
不确定性 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#465] [General]
# 不确定性 - Miao Yu | 于淼

https://yufree.cn/cn/2024/05/06/uncertainty/

**Comment by earfanfan** [comment]
三水，你好，你这“添油加醋的演绎”我读了以后产生了一种奇怪的感觉，大概就是，人类是有限的，人的个体经验是有限的，人的脑容量是有限的，人的知识和思考也是有限的，每个人都可以从世界上抓到一些东西，然后经由自己的问题意识、个体经验、思维逻辑加工整理后，梳理出来一些新的逻辑，但还是有限的。在有限的前提下，当你想要褒奖不确定性时，就可以将不确定性和确定性放在一个统一的标准里去比较，此消彼长之下便能突出其中之一的优势。为了支撑论点而找来的论据，也是有限的。比如“宗教就特别喜欢理性的人，甚至中世纪有段时间就是宗教理性保留并传承了不少科学知识”，如果去掉理性，这里本来呈现出来的事实就是中世纪某段时间由宗教保留并传承了不少科学知识，如果这一点作为论据去支撑其他论点似乎也是可行的，比如，很多。单单针对“宗教”和“理性”，如果非要推翻这个论点的话，也可以找出许多别的论据。

好了，我写上面这段并不是想要反驳你什么，其实应该是又要向你请教一个问题。我最近读了两本文学书，一本是博尔赫斯的《杜撰集》，另一本是《一个无政府主义者的意外死亡》（PS这两本你该不会又刚好都看过吧？），读的时候都为故事中的多重反转深感震撼，读完以后思考就陷入了一种奇妙的混乱。我感觉真实好像不太真实，很多事情（很多数据）都可以换不同角度去解读，以前我是选一个方向，然后并不排斥其他方向，尽量保留质疑空间，现在变成不知道选哪个方向了。啊，大概就是思绪变得混乱以后，发散出来很多个方向，没能捋出来一条主线。

话又说回来，当初三水你为撒会走上科研的路呢？

**Reply by yufree** [reply]
我只知道博尔赫斯的无限图书馆，没看过你说的。我不清楚思绪为啥需要主线，混乱也是一种正常的状态，硬说主线，现代社会里大概就是宪法，而且这到底不过是些形而上的概念辨析，而概念辨析很多时候没啥实际意义，又不影响第二天菜价。
我走科研没啥特别原因，可以说小时候志向（确实可能有），也可说高人指点（也确实可能有），还可以说养家糊口（也没错），甚至梦里被未来穿越者暗示操纵（也有可能），这样的原因只要我想找能找到很多，甚至都是真的，只看具体场景我需要用哪个就说哪个。我没必要找出个唯一或唯几原因，不是每件事都需要绑定到叙事逻辑里被完全解释，天天去讲因为所以，追求因果可以说理性，某种程度也可以说愚蠢，要允许脑子里存在想不开才能想开。

**Reply by earfanfan** [reply]
>某种程度也可以说愚蠢

三水你这半句话又戳中我莫名奇妙的笑点，哈哈，今天一天想起来一次乐一次。话说你可能还记得，多年前的一天或好几天，你跟杨洵默酣畅淋漓地辩论过世界是变得更好还是变得更坏。当然，我也在互联网上留下无数的痕迹，在相信所见所得的那一面是唯一真理的阶段，感觉以前的自己虽也算是某种程度的愚蠢，但那种坚定的相信其实也还挺有趣的。

我知道脑中思绪混乱是一种常态，可是当需要说出来或者写下来的时候还是需要有逻辑的。前面留言的时候我没写清楚一些背景，那这里展开写写哈。需要提前说明的是，其实我并没指望你来帮我解决我的问题，只是借与你的交流捋一捋自己的思路，所以你不用有撒想要帮别人忙的心理负担。事情是酱紫的，我以前写博客比如以一个主题展开思考或者以一个方向收集数据开始分析，完成的过程中总是有一条主线的，换句话说是有目标的，尽管一边写也会一边怀疑，但是继续写与继续怀疑是保持着一种平衡的状态的，所以都能做完。后来继续写和继续怀疑之间开始变得有些失衡，这种新的状态下，尽管会从主线发散出来一些支线，可是我为了写完会做些取舍。可是现在失衡变得……变得没法只是做些简单的取舍了。

额，可能我还是没写清楚到底是撒情况。就以你们曾经辩论过的世界是变得更好还是更坏为例子吧，我发觉思考和分析变得简单了，我从看过的书里借到了很多分析的方法啊框架之类的东西，这个题目可以发散出来很多思路，比如宏观微观的角度啦，国家比较的角度啦，各种指标比较的角度啦，个体经历演绎的角度啦，动态变迁的角度啦，总之可以发散出来很多的方向，可是我似乎选择不了，当要去写的时候，该用什么逻辑把各种论据串起来推导出一个结论，或者选择其中哪些论据去支撑一个有倾向性的结论，又或者说该怎么忍住不发散。或者说，要通过比较来分辨事物的优劣时总得有个用于衡量的标准，可是一旦面临继续写的选择时，我脑中混乱的思绪就会被许多个不同的标准困住，即便写出来了一些，也会迷惑于写到哪里才算完。

当然，正如你所说，即便思绪混乱也根本与日常生活无关，反正我的时间已经被工作和吃饭睡觉遛狗给排好了秩序。另外，我也没把这种“混乱”当做一个困难或者问题，即便思考和分析写不出来了，还是能继续耕点感受生活类的博客的。好吧，我暂且相信你说的“要允许脑子里存在想不开才能想开”，虽然不知道我是具体哪里想不开，不过就这么当做吧。

无限图书馆是不是还有别的译名，我去年看的博尔赫斯的《小径分岔的花园》里有一个短篇名为《通天塔图书馆》。

**Reply by yufree** [reply]
我笔记里写废了的片段比放出来的只多不少，很多博客文章文不对题也很常见，不过作为计划生育一代的独生子女，对着空气说话那属于无师自通的技能，写博客不过是把空气换成网页而已，现在要是哪天把我跟AI的聊天记录放出来，估计又是个奇葩人类的观测样本了。写作这东西，写给别人是要考虑逻辑主题的，写给自己怎么舒服怎么来，当成情绪出口也无所谓，当然得考虑下要不要公开了。

不论写给谁，想让自己“客观”或“中立”都是徒劳的。所谓摆事实讲道理，摆同样的事实讲不同的道理那是政客与律师玩弄概念的后院，用同样的道理去解释不同甚至矛盾的事实则是巫祝教主们的必修课，他们都需要学习逻辑与理性，这里逻辑与理性都是工具，真相就如同波粒二象性一样可能完全不讲理，这种没法讲理甚至推翻概念定义的事实与规律我们大都称之为科学。因此，科学需要逻辑、理性来实现客观与中立，但人不能装着科学去实现客观与中立，当设想自己列举某个主题所有观点时，逻辑上这个主题已经没有进一步讨论空间了，但科学总是要给未知留空间的，尽管这经常成为不可知论者攻击科学的一种手段。自己的写作对得起自己就可以了，觉得不完美就不发，这根本就不是一个需要纠结的问题。

另一个问题就在于你定义的“客观”或“中立”可能在另一个人看来可能漏洞多的跟筛子差不多。不是说你考虑不周全，更可能是你没法兼容本质矛盾的三观，跟观众能交流的只有你们知识与世界观的交集，这个交集对有些作者或有些读者本就不大，不如互道傻叉，喝杯酒就此别过，只要你还有个自认广阔可探索的世界就可以了，有机会下次在另一个地方跟之前的傻叉再聚再聊，当然你在他眼里大概率也是傻叉。这个世界充满了各种傻叉与荒谬但还是能运行，包容性还是有的，为傻叉干杯（免责声明：饮酒有害健康）。

确实是《通天塔图书馆》。

**Reply by earfanfan** [reply]
嗷，伸爪，干杯，俺干活去也。

**Reply by earfanfan** [reply]
三水啊，我忽然有点回过味来了，你说“不如互道傻叉，喝杯酒就此别过”好像不适用于现在的我，我生活的这片世界和你生活的那片世界很不一样，我一直在学习让自己更有同理心一点，嘴上不用自己的标准去审判别人，就连心里也尽量如此。

下面是我最近收集到的一些样本，如果要打算写到自己的博客的话，会陷入“混乱”状态很快就写不动，但是写到你的园子里好像是写得动的。不记录就会忘记真是浪费。所以如果你有兴趣看的话，不妨看下去，没兴趣看就放着不管吧。

我因为半句“某种程度也可算是愚蠢”总是乐得发笑，堂妹依依因为我笑得开心也跟着笑，笑着笑着我给依依描述了一些“三水与人辩论时通常战斗力很强”之类的，依依吐槽我的反应像是“赛博点笑穴”，然后讲了一个故事。话说依依上初中的时候班主任对早读都管得很严，迟到了是会被责罚的，有一天早读的时候她刚去教室就看到一排男同学面对教室的墙壁跪在地上，这么大阵仗真得很难忍住不好奇是发生了什么，原来是那班男同学前一晚在宿舍里争论一个议题，很晚都不睡觉，并且把好几层楼的人都吵醒了，他们争的是“到底是中国更屌还是美国更屌”。

人与人可以交流的并不只有知识与世界观的交集呐，还可以有各自生活中的所见所闻所感，或者理解与支持之类的。比如我表姐青青小学毕业就出去打工了，二十几年来都是早上八点上班晚上十二点下班，她居然很少抱怨工作时长太长，每每我说起打工很辛苦之类的，她的反馈却都只有一些无奈和无力改变，以及羡慕我能够五点半下班，前几天她跟我说起邻村的一个人五十多岁了为了挣钱还总加班到凌晨一两点，而且还是心脏动过手术的，几天前那人工作中睡着了就没有醒来。如果说只把青青和她提到的那个人当做观察样本的话，我暂时无法把收集到的样本拿来用于总结出什么结论。对了，我发现如果我真的从心里就没有用自己的标准审判别人，当我愿意倾听的时候，就可以倾听到很多与我思想观念不同的人所说的故事。话又说回来，有一天我认可了自己的认可欲求，与人交流既可以说是寻找共鸣，也可以说仅仅只是一种交流，就是字面意义上的交换和流通。

如果确实如你所说，我想要追求“客观”和“中立”，是说得通的。我通过生活中接触到的样本总结出一些结论，这个结论就是无法是全面的，无法去概括或者说去容纳整体所有样本。不过其实我纳闷了一下，为撒你会讲到“客观”或“中立”呢，我此前在你的园子里留言好像从未提过这两个词，发散一下，这是你曾经面对过类似问题后得到的解法还是你思考分析后得到的答案呢。按照历史经验，你传授的解法或者答案通常都很有用，比 AI 给的还好。已经是凌晨一点半，写到这里我忽然想了一下，我到底是把三水你当做一个人类还是一种功能呢，理论上我应该会活得比你久，未来某一天等你的人类身体死了以后，我可以把你留下的文字训练成一个人工智能，然后三水就真的变成功能了，哈哈哈。

**Reply by yufree** [reply]
不用等那么久，OpenAI的GPTs插件已经有这个功能了，只要把我写过的东西发过去就能生成赛博版来进行问答，只是你只能看到我公开的记录。

**Reply by earfanfan** [reply]
>不用等那么久

其实我没有在等。容我先拍一下你的马屁，啊，三水，我觉得世界上有你这个人类的存在真是太好了呀，三水人类的存在和三水 AI 的存在是不一样的。对我来说，三水 AI 的生成场景本来是用来替代三水人类的，如果要提前的话，从用户角度看，我觉得替代不了。

其一，语言文字对真实世界的描述是有限的。举个例子，如果说我们见到一处自然风景从而感到心旷神怡，这里有两次“信息量的压缩”（PS这里想不到撒更合适的词语了就编一个先），第一次信息压缩发生在用语言描述真实世界，语言本身是工具，为了能够使工具本身能够更清晰地描述世界，很多细节都有了特定的名称，又为了更准确第描述世界，在名称之上又添加了许多形容或者衡量的标准，比如一阵风是名称，微风、大风是形容风的，倘若要把用感知到的自然风景的一切视觉、触觉、嗅觉全描述出来会花费很多文字，于是有了第二次信息压缩，就是有些固定的词语、成语、短语与固定意象绑定，比如心旷神怡、千头万绪等等，用更少的文字表达更多。有了这样多层次的信息压缩，当你写下一些文字时，就算我看你博客好些年也未必能够很准地解压缩，所以我怀疑 GPTs 现在的能力在解压缩时也会失误。

其二，三水你在互联网世界公开留下的文字，也只是你的一部分。我在思考的时候，总是很难避免自我怀疑和自我否定（PS不是说这样就一定不好），如果把我的这点个人经验也迁移应用到你的身上，也许你在思考的时候也并不是永远做到自我肯定的，但是你写出来的文章是会把怀疑的过程去掉或者大幅简化最终留下肯定的，如前面讨论的，思绪在脑中是混乱的，写下来要运用语言文字这个工具的时候才是需要用逻辑去约束的。所以三水 AI 只会是三水的一个版本。

其三，训练一个三水 AI 也跟训练机器学习模型一样有个微调的过程，如果是我来使唤 AI 帮我训练三水 AI 的话，微调这一步我就觉得会调着调着跑偏，因为我也不知道训练这个干嘛用的，或者说不知道最应该保留或者最想要的特点是撒。要是现想的话，总结起来就是这些，三水人类很有耐心，最开始给你留言的时候我的逻辑时常是偏于混乱的，有些思绪自己都觉得难以捕捉然后清楚地描述出来，但你居然看了以后还能根据你的理解再给我展开讲讲，三水人类让我充分地感受到自己被当做一个平等独立的人类个体来尊重，没有爹味，三水人类的沟通方式简单直接，啊，这点真是太美好了，这几点估摸三水 AI 都能做到。不过我觉得三水 AI 做不到的应该就是三水你理性的表达中包含的一种“人味”，额，大概是一种同理心，这点要是从我的感知来讲，就是一些事情，就不长篇大论展开了哈。另外，我有订阅你的博客，这几年看下来，感觉到三水人类有许多细微的变化，我称之为进步，三水 AI 训练好了以后咋让它进步也是个问题。

其四，这点我没想太明白，大概是酱紫的，你应该记得以前 COSX 坛子里讨论过一篇文章，大概就是如果用模型描述世界的话，是数学模型更好还是机器学习模型更好之类的。文章具体讲的撒我记不太清了，以现在的记忆来解读的话就是，认同数学模型更好的，也认同世界上存在不可推翻或者不可证伪的科学定理，可以用数学模型来描述世界，不确定的是未必那些数学模型都已经全部被人类发现了，就算一套数学模型被推翻了，还可以建立新的；而认同机器学习模型更好的，基本思想偏向于统计学里面概率论那套，就是收集足够多的样本去估计误差分布，即在肯定一部分确定性的同时使用计算机工具去衡量不确定性。如果训练出来三水 AI ，其背后就是超大量的文本样本和超多参数的大模型，偏向后者。其实几年前我刚读那篇文章的时候也是有明显倾向的，但我现在只是产生了质疑的感觉，却无法清晰地表达出来质疑了什么，总觉得是因为我本身专业学的是统计学，就在这个框架下思考就会自然倾向后者，可是不管数学模型还是机器学习模型都是存在于让人们相信了一些科学定理存在或者统计学工具有用的前提条件下的，我可能是在质疑这些“相信”的真实，总而言之，相比于三水 AI，我会更愿意相信三水人类。

**Reply by earfanfan** [reply]
还有两个小点，三水 AI 是做不到的，一是三水人类时常让我感觉到这个人很坚强，二是有时候一本正经地描述刚好能戳中我的笑点。如果说 AI 很坚强，好像哪里怪怪的。

**Reply by yufree** [reply]
窃以为人类对于现象机制的理解跟实际情况是有距离的，自以为的“知道”可能只是真相的一小部分，当事物过于复杂时甚至人的认知水平能否理解都成问题。因此我不好评价你的观点，AI能不能做到完全仿真我也不关心，好比钻石在工业应用上是否是人造根本无所谓。另外，我看到过很多现实不认识我基于我的文字对我的评价，很多读下来我觉得是在描述另一个人，混合了很多逻辑上合理但并不真实的想象，不过无所谓，如果能从我的文字里体会到什么道理，哪怕跟我本意是反的也不重要，很多时候我自己都不知道我在想什么，经常有“原来我这么说是这个意思啊”的神奇感觉。科学哲学里曾有个实在论与非实在论的月亮辩论，所有人都看到月亮了所以其是存在的，但因为潮汐锁定没人看到过月亮背面，因此你可以把背面想象成任何形态也无法验证，那个时候其存在与否只能是一种待检验的信念而非真实，甚至会有人质疑所谓的检验也是片面的，有实用价值但无关真实。当然，现在登陆背面就基本消除这个想象空间了。

**Reply by earfanfan** [reply]
三水你居然也会“窃以为”，哈哈，以前我总忍不住把你当做一个老师的身份来看待，看样子从今天起可以只把你当做一个平等独立的人类朋友来看待了。如果哪天你觉得好评价了，也可以回来评价一下，这个都随你。

你说的这种神奇感觉，我体会过两种不同的。第一种是网络上的交流，举个例子，以前你写了一篇名为评分的博客，当时我有些留言，由于我也没能一次性说清楚自己的想法，然后其他小伙伴以为我是想强行说服你什么，我自己觉得脑子里好像没有出现过这样的念头，不过回头细读，啊，原来看上去确实很像是在这么干。这种只能说，语言文字作为传递信息的载体，有误解也很正常，有些时候大家看到一些文字或许提前预设了一些条件，已经作了善意或恶意的假定。不过换个角度想想，要是有一天，一个人脑子里不管想到什么或者表达什么，全都能百分百传递给别人，这一方面很可怕，另一方面却又很无趣了吧。即便技术上能实现，感觉还是会回归到不能够百分百传递的状态。

第二种是现实中的，也举个例子。前几天端午节我租了个车开回去把阿木狗也带回去了，由于阿木狗是绝育的公狗，所以总是会被村里别的公狗误以为是母狗跟着它。有天晚上我跟堂妹依依在门口马路上走，阿木狗跟着我们，经过一户人家的时候，那家的黑狗就悄悄跟着阿木狗，我听着后面一直有狗爪子触地的声音就突然回头一看，在这电光火石之间呐，黑狗迅速调整身体的方向，九十度转向，直直地盯着路边的河看，我当时没明白为撒黑狗要假装看河，于是继续盯着它看，这下子黑狗看河也看不下去了，于是身体保持不动继续转动狗头背向我，以此远离我的目光。当时堂妹依依没注意到这件小事，我就讲给她听，她听了笑着吐槽“原来狗也会尴尬啊”。我瞬间明白，原来黑狗是觉得尴尬啊，对，原本我想说的就是依依吐槽的，只是自己之前也不知道。

你最后说的这点好像又说回到博客主题了，认为真实或者认为虚假，认同确定性或者认同不确定性，都只是人们的价值观念，价值观念不同的人自然身怀不同的信念做出不同的行为，不过这就暂不继续讨论了，我现在力所不能及。

最后的最后，从我的视角看，我看待你的文字并不像是混合不真实的想象，应该说我描述的你就是你留在我脑中的印象，一般来讲是留下了丰富的印象，只不过随时间流逝，我只记得印象最深刻的那一部分特点，当然，这一部分理所应当在我脑子里是加工过的。所以说嘛，你还活着，我要训练撒三水 AI 咧。

**Reply by earfanfan** [reply]
>窃以为人类对于现象机制的理解跟实际情况是有距离的，自以为的“知道”可能只是真相的一小部分，当事物过于复杂时甚至人的认知水平能否理解都成问题。因此我不好评价你的观点……

三水啊，我忽然有点明白你之前说的撒了，我之前在思考要使用人工智能来生成三水 AI的时候，用的是一种生产产品的思维，这是有问题的，因为人的脑子，在工作效率、思维模式、容量、使用方式上全都受着人类身体、社会文化、当前时代科技发展水平等限制，但是机器本来是可以突破这些限制的。所以我猜，也许人工智能并不是只能发展成机器模拟人这个方向。可是突破人类的那些限制后，机器的智能可以怎样发展现在我也想象不出来。

**Reply by yufree** [reply]
原来我说的是这个意思啊，受教了。
